lua/terrortown/lang/en.lua

```diff
@@ -1545,6 +1546,10 @@ L.help_detective_hats = [[
 Policing roles such as the Detective may wear hats to show their authority. They lose them on death or if damaged at the head.
 
 Some player models do not support hats by default. This can be changed in 'Administration' -> 'Player Models']]
+L.help_inspect_credits_always = [[
+When players with credits (detectives, traitors, survivalist, etc.) die, their credits can be picked up by other players that can use credits. 
```

Comment:
We tend to call them "shopping roles" in code and documentation, maybe you can change the wording a bit here